PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionAdvancedConfigDialog.java

```diff
@@ -118,167 +184,89 @@ private void addRemoteDebugLineItem() {
-                    sshPasswordUseKeyFileTextField.setEnabled(true);
-
-                    ButtonModel currentSelection = sshPasswordButtonGroup.getSelection();
-                    sshPasswordUsePasswordRadioButton.setSelected(true);
-                    sshPasswordUseKeyFileRadioButton.setSelected(true);
-                    currentSelection.setSelected(true);
-                }
-                else
-                {
-                    sshUserNameTextField.setEnabled(false);
-                    sshUserNameLabel.setEnabled(false);
-                    sshAuthTypeLabel.setEnabled(false);
-
-                    sshPasswordUsePasswordRadioButton.setEnabled(false);
-                    sshPasswordUseKeyFileRadioButton.setEnabled(false);
-
-                    sshPasswordUsePasswordField.setEnabled(false);
-                    sshPasswordUseKeyFileTextField.setEnabled(false);
-                }
-            }
-        });
-
         add(sshAuthTypeLabel,
                 new GridBagConstraints(0, ++displayLayoutCurrentRow,
                         0, 1,
                         0, 0,
                         GridBagConstraints.WEST, GridBagConstraints.NONE,
-                        new Insets(margin, margin, 0, margin), 0, 0));
+                        JBUI.insets(margin, margin, 0, margin), 0, 0));
 
         add(sshPasswordUsePasswordRadioButton,
                 new GridBagConstraints(0, ++displayLayoutCurrentRow,
                         1, 1,
                         0, 0,
                         GridBagConstraints.WEST, GridBagConstraints.NONE,
-                        new Insets(margin / 3, margin * 3, 0, margin), 0, 0));
+                        JBUI.insets(margin / 3, margin * 3, 0, margin), 0, 0));
 
         add(sshPasswordUsePasswordField,
                 new GridBagConstraints(1, displayLayoutCurrentRow,
                         0, 1,
                         1, 0,
                         GridBagConstraints.WEST, GridBagConstraints.HORIZONTAL,
-                        new Insets(margin / 3, margin, 0, margin), 0, 0));
+                        JBUI.insets(margin / 3, margin, 0, margin), 0, 0));
 
         add(sshPasswordUseKeyFileRadioButton,
                 new GridBagConstraints(0, ++displayLayoutCurrentRow,
                         1, 1,
                         0, 0,
                         GridBagConstraints.WEST, GridBagConstraints.NONE,
-                        new Insets(margin / 3, margin * 3, 0, margin), 0, 0));
+                        JBUI.insets(margin / 3, margin * 3, 0, margin), 0, 0));
 
         add(sshPasswordUseKeyFileTextField,
                 new GridBagConstraints(1, displayLayoutCurrentRow,
                         0, 1,
                         1, 0,
                         GridBagConstraints.WEST, GridBagConstraints.HORIZONTAL,
-                        new Insets(margin / 3, margin, 0, margin), 0, 0));
+                        JBUI.insets(margin / 3, margin, 0, margin), 0, 0));
```

Comment:
May be we can provide the link of resetting SSH password here?

Reply:
I didn't find a good placement for resetting SSH password doc hyper link here. Since there is already a big change here. I suppose to add it later.